nonlocal-warning.scala:6: warning: This catches all Throwables. If this is really intended, use `case x : Throwable` to clear this warning.
    catch { case x => 11 }
                 ^
nonlocal-warning.scala:4: warning: catch block may intercept non-local return from method foo
  def foo(l: List[Int]): Int = {
                               ^
error: No warnings can be incurred under -Werror.
2 warnings
1 error
